packages/babel-jest/src/index.js

```diff
@@ -102,16 +105,23 @@ const createTransformer = (options: any) => {
       src: string,
       filename: Path,
       config: ProjectConfig,
-      transformOptions: TransformOptions,
-    ): string {
+      transformOptions?: TransformOptions,
+    ): string | TransformedSource {
       const altExts = config.moduleFileExtensions.map(
         extension => '.' + extension,
       );
+      const shouldUseInline =
```

Comment:
as mentioned, inline sourcemaps are useful for IDEs and debugging, so I don't think they should be removed

Reply:
We will revert to using `both`?

Reply:
I can't think of a reason not to :)